mage_ai/frontend/components/Backfills/Table/index.tsx

```diff
@@ -88,7 +89,17 @@ function BackfillsTable({
         total_run_count: totalRunCount,
       }, idx) => {
         const arr = [
-          <Text default key="status" monospace>{status || 'inactive'}</Text>,
+          <Text
+            danger={BackfillStatusEnum.FAILED === status}
+            default={BackfillStatusEnum.CANCELLED === status}
```

Comment:
`default` prop when `status` is null. This is a greyed out color where `inactive` status is displayed in the UI.